docs/dashboards/layout-description.md

```diff
@@ -97,4 +72,63 @@ Example:
         flex: 1 1 100%;
     }
 }
-```
\ No newline at end of file
+```
+
+Instead of regular CSS media queries, you can also use [container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries), which will make more sense if your dashboard does not cover the entire window horizontally.
```

Comment:
None of our demos has it, maybe let's add a simple demo with it?

Reply:
The accounting demo has it 😃 I will add a link to it.

Reply:
I can have a look at it.